bug_update.php

```diff
@@ -154,24 +154,25 @@
 if ( !$t_reporter_reopening && !$t_reporter_closing ) {
 	switch( $f_update_type ) {
 		case BUG_UPDATE_TYPE_ASSIGN:
-			access_ensure_bug_level( 'update_bug_assign_threshold', $f_bug_id );
+			$t_threshold = 'update_bug_assign_threshold';
 			$t_check_readonly = true;
 			break;
 		case BUG_UPDATE_TYPE_CLOSE:
 		case BUG_UPDATE_TYPE_REOPEN:
-			access_ensure_bug_level( 'update_bug_status_threshold', $f_bug_id );
+			$t_threshold = 'update_bug_status_threshold';
 			$t_check_readonly = false;
 			break;
 		case BUG_UPDATE_TYPE_CHANGE_STATUS:
-			access_ensure_bug_level( 'update_bug_status_threshold', $f_bug_id );
+			$t_threshold = 'update_bug_status_threshold';
 			$t_check_readonly = true;
 			break;
 		case BUG_UPDATE_TYPE_NORMAL:
 		default:
-			access_ensure_bug_level( 'update_bug_threshold', $f_bug_id );
```

Comment:
Provide the project id in the access check since current project may not be the same as the project associated with issue being edited.

Reply:
@vboctor this should not be needed as we have a "nice" way some lines above to deal with it.
```
if( helper_get_current_project() != $t_existing_bug->project_id ) {
	$g_project_override = $t_existing_bug->project_id;
}
```

Reply:
Makes sense.